src/Orchard.Web/Modules/Orchard.OpenId/Controllers/AccessController.cs

```diff
@@ -4,18 +4,19 @@
 using AspNet.Security.OpenIdConnect.Server;
 using Microsoft.AspNetCore.Authentication;
 using Microsoft.AspNetCore.Authorization;
-using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Http.Authentication;
 using Microsoft.AspNetCore.Identity;
 using Microsoft.AspNetCore.Mvc;
 using OpenIddict;
+using OpenIddict.Mvc;
+using Orchard.Mvc;
 using Orchard.OpenId.Services;
 using Orchard.OpenId.ViewModels;
 using Orchard.Users.Models;
 
 namespace Orchard.OpenId.Controllers
 {
-    [Authorize, ValidateAntiForgeryToken]
+    [Authorize, AutoValidateAntiforgeryToken]
```

Comment:
> Or is that an extraneous safety level to ensure it's really there.

Exactly that. I'm paranoid :smile: 
If you think it's not worth it, I'm fine removing it.